etc/profile-m-z/tar.profile

```diff
@@ -7,13 +7,17 @@ include tar.local
 # Persistent global definitions
 include globals.local
 
-# Arch Linux (based distributions) need access to /var/lib/pacman. As we drop all capabilities this is automatically read-only.
-noblacklist /var/lib/pacman
-
+# From archiver-common.inc
```

Comment:
^

Reply:
My bad, I had only seen it on 7z.profile.  Now it's changed everywhere.